
# python使用traceback获取详细的异常信息
```
 except:
        #抛出错误信息
        traceback.print_exc()
```

# python #!/usr/bin/python作用

直接执行(并给予权限， ./test.py)时，没指定会报syntax error near unexpected token `


# Python 使用 raise 语句抛出一个指定的异常。
raise Exception('x 不能大于 5。x 的值为: {}'.format(x))

